core/server/models/post.js

```diff
@@ -68,6 +68,13 @@ Post = ghostBookshelf.Model.extend({
         posts_meta: 'posts_meta'
     },
 
+    relationsMeta: {
+        posts_meta: {
+            targetTableName: 'posts_meta',
```

Comment:
Key names `targetTableName` and `foreignKey` were borrowed from naming used in bookshelf models internally